src/components/AoInfoPair.vue

```diff
@@ -18,6 +24,10 @@ export default {
     label: {
       type: String,
       required: true
+    },
+    labelHtml: {
+      type: String,
+      required: true
```

Comment:
making it a required prop may not be great as it would mean changing wholesale and S2S to include this with empty value

Reply:
`label` and `label-html` are optional now